pipeline-scripts/release.groovy

```diff
@@ -117,11 +117,15 @@ def getArchSuffix(arch) {
 
 def stageGenPayload(dest_repo, release_name, dest_release_tag, from_release_tag, description, previous, errata_url) {
     // build metadata blob
-    def metadata = "{\"description\": \"${description}\""
-    if (errata_url) {
-        metadata += ", \"url\": \"${errata_url}\""
+
+    def metadata = ''
+    if (description || errata_url) {
+        metadata = "{\"description\": \"${description}\""
```

Comment:
Might be easier with [`JsonOutput`][1]?  I'm trying to work up code for that approach...

[1]: https://groovy-lang.org/json.html#_jsonoutput

Reply:
Something like:

```groovy
def metadata = {}
if (description) {
  metadata.description = description
}
if (errata_url) {
  metadata.url = errata_url
}
```

and then later:

```groovy
if (metadata) {
  cmd += "--metadata '" + JsonOutput.toJson(metadata) + "' "
}
```

or some such?

Reply:
just because it's in groovy doesn't mean using it in a pipeline is easy. jenkins sandbox makes it really hard to do anything sane with json.